pysat/version.txt

```diff
@@ -1 +1 @@
-2.2.2
+3.0
```

Comment:
```suggestion
3.0.0
```

Keep up the same semantic versioning style

Reply:
I did look up semantic versioning here ahead of time, https://www.python.org/dev/peps/pep-0440/.
In this examples before I saw use of only `x.0`. This time around I saw 
```
Note
X.Y and X.Y.0 are not considered distinct release numbers, as the release segment comparison rules implicit expand the two component form to X.Y.0 when comparing it to any release segment that includes three components.
```
Happy to expand it back.

Reply:
Please do, since we've expanded everything out for the other pysat Ecosystem packages 🐳